index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=375px, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+</head>
+<body>
+  <div class="header">
+    <div class="welcome miss-desktop">Welcome</div>
+    <nav class="nav">
+      <a href="#" class="menu miss-desktop"><i class="material-symbols-rounded">menu</i></a>
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>  
+  </div>
+  <main>
+    <section class="headline">
+      <div class="item">
+        <h2 class="intro">Hey there, I'm Harriet</h2>
+        <h2 class="intro green">I'm a software developer</h2>
+      </div>
+      <p class="item"> 
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socials item">
+        <i class="fab fa-brands fa-twitter"></i>
+        <i class="fab fab-brands fa-linkedin-in"></i>
+        <i class="fab fa-brands fa-github"></i>
+        <i class="fa fa-regular fa-hand-peace"></i>
+      </div>
+      
+      <button class="arrow item"><i class="fa fa-solid fa-chevron-down"></i></button>
+      <h2 class="item">My Recent Works</h2>
+      <hr/>
+    </section>
+    <section class="card grid">
+      <div class="item img-holder grid-item-1">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-2">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-3">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
```

Comment:
- Kindly, when the previous code reviewer suggested using `hx` means `h1`-`h6` :+1: :+1:  update it because `hx` unknown tag